tests/__compound-components__/Filters/Facets.tsx

```diff
@@ -1,8 +1,9 @@
 import { Filters } from '../../../src/components';
 
-export function Facets({ searchOnChange = true }: { searchOnChange?: boolean }): JSX.Element {
+export function Facets(args: Filters.FacetsProps): JSX.Element {
+  const searchOnChange = args.searchOnChange === false ? false : true;
```

Comment:
nit: maybe we can have a defaultConfig object so it's clearer? (`{...defaultConfig, ...args}`)

Reply:
Since there's only one default option (`searchOnChange`), I updated to merge it with `args` directly